src/express/api.jl

```diff
@@ -126,3 +144,123 @@ end
 function activate(runnable_simulation_code::Function, container::ContainerInterface)
     activate(runnable_simulation_code, [container])
 end
+
+"""
+    run_in_real_time(runnable::Function,
+    n_container::Int,
+    container_list_creator::Function,
+    agents::Union{Tuple,Agent}...)
+
+Let the agents run in containers (real time).
+
+Distributes the given `agents` to `n_container` (real time container) and execute the `runnable` 
+(takes the container list as argument) while the container are active to run. It is possible to 
+add supplementary information per agent as Tuple. For example `(Agent, :aid => "my_aid")`.
```

Comment:
A tuple mapping symbols to values... so a dict (as you directly cast it to below as well). 
Is there a reason not to pass it as a dict to begin with then?

I think a construct like this may end up being confusing once people are also used to the new  `getindex` on agent... because `agent[1]` is something entirely different to `agent[1]` in other contexts now:
```julia
        if isa(agent, Tuple)
            actual_agent = agent[1]
```

Is there a nicer way to do this that doesn't involve much overhead? It's obvious what it does when one keeps the docstring in mind but I just wonder if this may result in some error down the line somewhere.